fetch.bs

```diff
@@ -2737,10 +2737,12 @@ with a <i>CORS flag</i> and <i>recursive flag</i>, run these steps:
  for <var>response</var>.
 </ol>
 
+<!-- Prior to May 2017, this algorithm was named "basic fetch." The
+     corresponding ID is maintained in order to preserve the functionality of
+     external references. -->
+<h3 id=scheme-fetch><span id=basic-fetch>Scheme fetch</span></h3>
 
-<h3 id=basic-fetch>Basic fetch</h3>
-
-<p>To perform a <dfn id=concept-basic-fetch for=basic>basic fetch</dfn> using
+<p>To perform a <dfn id=concept-scheme-fetch>scheme fetch</dfn> using
```

Comment:
Add `oldids=concept-basic-fetch` here too. I suspect both of these to have external links.